packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -202,6 +199,59 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
         }
         dataStoreConfigurationBuilder.errorHandler(errorHandler)
 
+        var conflictHandler: DataStoreConflictHandler
+        conflictHandler = if ((request["hasConflictHandler"] as? Boolean? == true)) {
```

Comment:
The double parentheses may be uncessary.

```suggestion
        val errorHandler = if (request["hasErrorHandler"] as? Boolean? == true) {
```

Reply:
changed to: 

`conflictHandler = if (request["hasConflictHandler"] as? Boolean? == true) {`